apps/examples/security_test/seclink/sl_auth_test.c

```diff
@@ -505,11 +505,164 @@ TEST_F(ecdsa_verify)
 	ST_END_TEST;
 }
 
+/*
+ * Desc: Get ED25519
+ * Referred https://developer.artik.io/documentation/security-api/see-authentication-test_8c-example.html
+ * This key pair is verified in http://ed25519.herokuapp.com
+ */
+uint8_t g_ed25519_privkey[] = {
```

Comment:
If `g_ed25519_privkey` and `g_ed25519_privkey_only` are used within this file, how about adding static?